Sources/Rendering/Core/CubeAxesActor/index.js

```diff
@@ -0,0 +1,791 @@
+import { vec3, mat4 } from 'gl-matrix';
+import * as d3 from 'd3-scale';
+import * as vtkMath from 'vtk.js/Sources/Common/Core/Math';
+import macro from 'vtk.js/Sources/macro';
+import vtkActor from 'vtk.js/Sources/Rendering/Core/Actor';
+import vtkBoundingBox from 'vtk.js/Sources/Common/DataModel/BoundingBox';
+import vtkDataArray from 'vtk.js/Sources/Common/Core/DataArray';
+import vtkMapper from 'vtk.js/Sources/Rendering/Core/Mapper';
+import vtkPixelSpaceCallbackMapper from 'vtk.js/Sources/Rendering/Core/PixelSpaceCallbackMapper';
+import vtkPolyData from 'vtk.js/Sources/Common/DataModel/PolyData';
+import vtkTexture from 'vtk.js/Sources/Rendering/Core/Texture';
+
+// ----------------------------------------------------------------------------
+// vtkCubeAxesActor
+// ----------------------------------------------------------------------------
+// faces are -x x -y y -z z
+// point 0 is 0,0,0 and then +x fastest changing, +y then +z
+const faceNormals = [
+  [-1, 0, 0],
+  [1, 0, 0],
+  [0, -1, 0],
+  [0, 1, 0],
+  [0, 0, -1],
+  [0, 0, 1],
+];
+const faceEdges = [
+  [8, 7, 11, 3],
+  [9, 1, 10, 5],
+  [4, 9, 0, 8],
+  [2, 11, 6, 10],
+  [0, 3, 2, 1],
+  [4, 5, 6, 7],
+];
+const edgePoints = [
+  [0, 1],
+  [1, 3],
+  [2, 3],
+  [0, 2],
+  [4, 5],
+  [5, 7],
+  [6, 7],
+  [4, 6],
+  [0, 4],
+  [1, 5],
+  [3, 7],
+  [2, 6],
+];
+const edgeAxes = [0, 1, 0, 1, 0, 1, 0, 1, 2, 2, 2, 2];
+const faceAxes = [
+  [1, 2],
+  [1, 2],
+  [0, 2],
+  [0, 2],
+  [0, 1],
+  [0, 1],
+];
```

Comment:
should probably be moved in a Constants.js file

Reply:
I don't want to export these ever (if they get exported it should be from a cell type class). So should they still be in a constants file? Just not sure what the motivation is for that case.

Reply:
I guess it is fine to leave it here if they do not make sense to be exported. @jourdain